lib/game.rb

```diff
@@ -0,0 +1,12 @@
+class Game
+  attr_reader :name, :cost, :national_drawing
+
+  def initialize(name, cost, national_drawing = false)
+    @name = name
+    @cost = cost
+  end
+
+  def national_drawing?
```

Comment:
Soooo close.  What we actually want to do is let the third argument (or lack of a third argument) set the instance variable, and then use this method to just return that instance variable:

```ruby
def initialize(name, cost, national_drawing = false)
  ...
  @national_drawing = national_drawing
end

def national_drawing?
  @national_drawing
end
```